test/e2e/app_management_test.go

```diff
@@ -737,3 +738,35 @@ func TestExcludedResource(t *testing.T) {
 		Then().
 		Expect(Condition(ApplicationConditionExcludedResourceWarning, "Resource apps/Deployment guestbook-ui is excluded in the settings"))
 }
+
+func TestOrphanedResource(t *testing.T) {
+	Given(t).
+		ProjectSpec(AppProjectSpec{
+			SourceRepos:       []string{"*"},
+			Destinations:      []ApplicationDestination{{Namespace: "*", Server: "*"}},
+			OrphanedResources: &OrphanedResourcesMonitorSettings{Warn: pointer.BoolPtr(true)},
```

Comment:
minor - I think you can do `&(true)`

Reply:
Unfortunately golang does not support it.